test/e2e/installplan_e2e_test.go

```diff
@@ -4028,8 +4029,182 @@ var _ = Describe("Install Plan", func() {
 		}).Should(HavePhase(operatorsv1alpha1.InstallPlanPhaseComplete))
 
 	})
+
+
+FDescribe("When an operator is attempted to be installed from a catalog", func ()  {
```

Comment:
Make sure to remove this `FDescribe` before we get this through, please.

Reply:
👍🏽 fyi the test will fail if the focus is on so no focused test can get in anyway.